Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteScrollingCoordinatorProxyIOS.h

```diff
@@ -75,6 +80,10 @@ class RemoteScrollingCoordinatorProxyIOS final : public RemoteScrollingCoordinat
 #if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
     HashSet<WebCore::PlatformLayerIdentifier> m_fixedScrollingNodeLayerIDs;
 #endif
+
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+    Vector<std::pair<RetainPtr<PlatformLayer>, Ref<RemoteAcceleratedEffectStack>>> m_effectsStack;
```

Comment:
I don't like retaining platform layers here. Can we store `PlatformLayerIdentifier` and look up layers on the fly?

Reply:
Why is this needed since RemoteLayerTreeNodes own the effects?

Reply:
Cc @mattwoodrow to answer, I believe he authored that part of the patch.

Reply:
Yeah, for iOS, I think we can do the obvious thing and just reference the Nodes.

Reply:
In the latest patch this class now has `HashSet<WeakPtr<RemoteLayerTreeNode>> m_animatedNodes` to track the nodes that have animations set on them.